Formula/goreleaser.rb

```diff
@@ -17,8 +17,9 @@ def install
     ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
     (buildpath/"src/github.com/goreleaser/goreleaser").install buildpath.children
     cd "src/github.com/goreleaser/goreleaser" do
-      system "go", "build", "-ldflags", "-X main.version=#{version}", "-o",
-             bin/"goreleaser"
+      system "go", "build", 
+             "-ldflags", "-s", "-w", "-X main.version=#{version}", 
```

Comment:
I think you meant:
```
         "-ldflags", "-s -w -X main.version=#{version}", 
```

But I don't see why the `-s` and `-w` are necessary. Per the `man ld` page, `-s` is obsolete, and `-w` just suppresses warnings, which I don't think we want. Has something changed between releases to motivate adding `-s -w`?

Reply:
is just what is used by goreleaser build itself... anyway, you're right, its not needed... will undo that..